configs/12.0/packages/gdb/sources

```diff
@@ -20,7 +20,7 @@
 #
 ATSRC_PACKAGE_NAME="GDB (GNU Debugger)"
 ATSRC_PACKAGE_VER=8.1.90
```

Comment:
I think this is correct, because, although we are tracking gdb-8.2-branch, GDB 8.2 has never been tagged (nor released), so there's no GDB 8.2, yet.

On the other hand, if we believe that it's not a good idea to release AT 12 with an unreleased version of GDB, we could/should switch to gdb-8.1-branch.

Reply:
Fair enough